package.json

```diff
@@ -174,7 +174,6 @@
     "node-sass": "^4.9.0",
     "null-loader": "^0.1.1",
     "nyc": "^11.1.0",
-    "path": "^0.12.7",
```

Comment:
We don't need to bring this is via NPM since `path` is native to Node and we aren't using this in the actual app, only in our tests which have access to Node.

Reply:
Sound good.